forest/config.py

```diff
@@ -59,7 +76,7 @@ def patterns(self):
         return []
 
     @classmethod
-    def load(cls, path):
+    def load(cls, path, variables=None):
```

Comment:
This doesn't need to be optional ... it looks like it's always passed in. In which case we can also remove the `if variables is not None` check below.

Reply:
It can be `None` as it comes from `args.variables` which is `None` when no `--var` flag is present. I've put unit tests in to cover that possibility

Reply:
It was optional from a open/closed principle perspective but I agree that it is now required to fulfil our new loading mechanism